app/controllers/concerns/hyrax/controller.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+# OVERRIDE: Setup an override on locale to render header
+module Hyrax::Controller
```

Comment:
See that this is a module & not a class. We should be able to find where this is imported and override the method there without pulling in this whole module

Reply:
https://github.com/osulp/Scholars-Archive/blob/main/app/controllers/application_controller.rb#L12